pkg/southbound/synchronizer/synchronizer.go

```diff
@@ -86,14 +90,14 @@ func New(context context.Context, changeStore *store.ChangeStore, configStore *s
 
 		if err != nil {
 			log.Error("Can't translate the initial config for ", sync.Device.Addr, err)
-			return nil, err
+			return sync, nil
```

Comment:
Would we ignore the error in every case?

Reply:
I expect this error never to happen, it's config we stored in the enos-config stores and should have been checked before, I'm still doing the check because I don't like ignoring error altogether thus the log is there but it's an error that we should never hit here.